flows/knack/knack_services.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/env python
+
+"""
+Name: ATD Knack Services
+Description: This set of tasks publishes data from ATD's various Knack 
+ applications to other platforms including a Postgres database, Socrata, 
+ and ArcGIS Online (AGOL). 
+Schedule: "30 14,18 * * *"
+Labels: atd-data02, knack
+"""
+
+import os
+
+import docker
+import prefect
+from datetime import datetime, timedelta
+
+# Prefect
+from prefect import Flow, task, Parameter, unmapped, case
+from prefect.storage import GitHub
+from prefect.run_configs import LocalRun
+from prefect.engine.state import Failed
+from prefect.schedules import Schedule
+from prefect.schedules.clocks import CronClock
+from prefect.backend import set_key_value, get_key_value
+from prefect.triggers import all_successful
+from prefect.tasks.docker import PullImage
+
+
+from prefect.utilities.notifications import slack_notifier
+
+
+# Default parameters is for Signs/Markings Contractor Work Orders
+FLOW_NAME_UNIQUE = "Markings Contractor Work Orders"
+APP_NAME = "signs-markings"
+CONTAINER = "view_3628"
+LAYER_NAME = "markings_contractor_work_orders"
+# Parameter that will overwrite all data (ignores date)
+REPLACE_DATA = False
+APP_NAME_DEST = ""
+SOCRATA_FLAG = True
+AGOL_FLAG = True
+
+# Define current environment
+current_environment = "prod"
+
+# Set up slack fail handler
+handler = slack_notifier(only_states=[Failed])
+
+# Logger instance
+logger = prefect.context.get("logger")
+
+# Select the appropriate tag for the Docker Image
+docker_env = "production"
+docker_image = f"atddocker/atd-knack-services:{docker_env}"
```

Comment:
same here, would the `docker_env` ever not be production?

Reply:
Refactored this into a parameter for our flow